packages/shared/lib/core/wallet/actions/getTotalWalletBalance.ts

```diff
@@ -1,13 +1,23 @@
 import { OutputData } from '@iota/sdk/out/types'
 import { IBalance, getBicBalance } from '..'
 import { getBalance } from './getBalance'
+import { getManaBalance, getTotalAvailableMana } from '../../network'
+import { getWalletById } from '../../profile'
 
 export async function getTotalWalletBalance(walletId: string, accountOutputs: OutputData[]): Promise<IBalance> {
     const balances = await getBalance(walletId)
     const blockIssuanceCredits = await getBicBalance(walletId, accountOutputs)
     const totalWalletBic = Object.values(blockIssuanceCredits).reduce((acc, bic) => acc + Number(bic), 0)
+
+    const _selectedWallet = getWalletById(walletId)
```

Comment:
This is going to break onboarding I think, `getTotalWalletBalance` is called in buildWalletState.ts`

Reply:
I haven't had any problems

Reply:
This will be avoided in https://github.com/iotaledger/firefly/pull/8404 anyway